server/app/views/admin/migration/AdminImportViewPartial.java

```diff
@@ -173,12 +202,27 @@ private DivTag buildQuestionAlert(
             alertMessage.concat(numNewQuestions + " new " + questionOrQuestions + " and ");
       }
       String questionOrQuestions = numDuplicateQuestions == 1 ? "question" : "questions";
-      alertMessage =
-          alertMessage.concat(
-              numDuplicateQuestions
-                  + " duplicate "
-                  + questionOrQuestions
-                  + " to the question bank.");
+      if (withDuplicates) {
+        alertMessage =
+            alertMessage.concat(
+                numDuplicateQuestions
+                    + " duplicate "
+                    + questionOrQuestions
+                    + " to the question bank.");
+      } else {
+        if (numNewQuestions > 0) {
+          alertMessage += " to the question bank. ";
+        }
+        String areOrIs = numDuplicateQuestions > 1 ? "are " : "is ";
+        alertMessage =
+            alertMessage.concat(
+                "There "
+                    + areOrIs
+                    + numDuplicateQuestions
+                    + " existing "
+                    + questionOrQuestions
+                    + " that will appear as drafts in the question bank.");
```

Comment:
thought (maybe blocking): It occurred to me that saying these questions "will appear as drafts" might be misleading because all of the imported questions will appear as drafts. I think an imported piece of information for the user will be how many programs contain this question and will be updated to be draft programs. But I defer to you on this since it will mostly be used by Charlotte and you will be helping them!

Reply:
Hmm yeah good point. In this PR, I decided to update all programs to draft when we import a program, since looping through and finding the ones that changed seemed like overkill and we make the user re-publish anyway. If we want to be more specific with the questions we put into draft, I can do it as a follow up. And I can definitely update this error message in the meantime, but unsure exactly what to say. Maybe something like "that will be updated to a draft state in the question bank"? That's pretty similar though - let me know what you think!

Reply:
Sounds good, thanks!